test/Yoti.Auth.Tests/UserProfileTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Yoti.Auth.YotiAttributeValue;
+
+namespace Yoti.Auth.Tests
+{
+    [TestClass]
+    public class UserProfileTests
+    {
+        [TestMethod]
+        public void UserProfile_GetAttribute_Datetime()
```

Comment:
It would be nice to have more descriptive names.
Ideally I would not need to read the body of the test to understand what's actually checked.
I usually use the form `testSomething` or the `shouldSomething`/`somethingShould`, which in this case would be something like `testParsedDateMatchesInitialValue` or `parsedDateShouldMatchInitialValue`

Reply:
I agree that more descriptive names would be useful so you can see exactly what the behaviour should be from the name along. Additionally, we don't need to prefix the tests with the class (`UserProfile` here), since that can be inferred from the name of the tests.

I would like the test naming to be consistent across the tests though, so if we are to rename them, I would like them all to be renamed that way. I have created a ticket for this - https://lampkicking.atlassian.net/browse/SDK-682